### PR TITLE
handle empty strings when totalling clin value

### PR DESCRIPTION
**Pull request by dandds**
Small JS fix for CLIN totals to address the issue noted here:

https://www.pivotaltracker.com/story/show/162772340/comments/198247235

### CODE REVIEW

File: js/components/forms/funding.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default {
     totalBudget: function () {
       return [this.clin_01, this.clin_02, this.clin_03, this.clin_04].reduce(
         function(acc, curr) {
-          curr = curr === null ? 0 : parseInt(curr)
+          curr = curr === null || curr === '' ? 0 : parseInt(curr)
```
Comment (montana-mil) [inline js/components/forms/funding.js]: could do `curr === (null || '')`, right?

Comment (montana-mil) [reply js/components/forms/funding.js]: or even `curr = curr ? 0 : parseInt(curr)`?

Comment (patricksmithdds) [reply js/components/forms/funding.js]: @montana-mil `curr === null || curr === ''` and `curr === (null || '')` are not equivalent. The second conditional would evaluate the `null || ''` part first so be equivalent to just `curr === ''`.

Comment (montana-mil) [reply js/components/forms/funding.js]: ok but just checking `curr` is ok, right?

Comment (dandds) [reply js/components/forms/funding.js]: @montana-mil yeah, I updated the PR. Just have to bang-invert it.

**Pull request merged**